src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -273,19 +279,26 @@ def get_model_compressor(
         format
     :param skip_compression_stats: bool allowing compression stats on std out
     :param state_dict: state_dict of the model
+    :param no_sparse_compression: bool to skip sparse compression
```

Comment:
I would add the table you listed in your PR description to the docstring and make sure the following cases are shown:

```
W8A16 weight only no sparse: naive
W8A16 weight + sparse: marlin24
W8A8 + no sparse: naive
W8A8 + sparse: sparse24

W8A16 FP8 weight only:  naive
W8A16 FP8 weight only + sparse: naive
W8A8 FP8 + no sparse: naive
W8A8 fp8  + sparse: sparse24
```